src/chrome/content/rules/KSL.xml

```diff
@@ -7,7 +7,13 @@
 	<securecookie host="^\.ksl\.com$" name=".+" />
 
 
-	<rule from="^http://((?:img|local|static|www)\.)?ksl\.com/"
+	<rule from="^http://ksl\.com/"
+		to="https://www.ksl.com/" />
+		<test url="http://ksl.com/" />
+	<rule from="^http://((?:img|local|static|www)\.)ksl\.com/"
```

Comment:
Please also add:
v2.ksl.com
studio5.ksl.com (mixed content, may need exclusions)
live.ksl.com (mixed content, may need exclusions)
videos.ksl.com (please check if this does not break streams on live or studio5)

Please document non-working ones on the top:
deals.ksl.com (HTTP redirect)
movies.ksl.com (mismatching cert)